speechAssets/IntentSchema.json

```diff
@@ -13,6 +13,15 @@
       "intent": "GetServiceAdvisories"
     },
     {
+      "intent": "SetHomeStation",
+      "slots": [
+        {
+          "name": "homeStation",
```

Comment:
Why not name this `station` to be consistent with the `GetStation` type? If this API were typed, they would be the same thing, no?